src/js/pages/scene-listener.jsx

```diff
@@ -190,7 +188,10 @@ var SceneListener = React.createClass({
           /*  console.log("state.scene changed - update player");*/
             this._maybeUpdatePlayer();
         } else if (!_.isEqual(prevProps.activeScene, this.props.activeScene)) {
-           /* console.log("activeScene changed - update player");*/
+            // APEP if we switch activeScene, we should make sure we unsubscribe for updates from previous scenes
+            // And resubscribe to the new active scene
+            HubSendActions.unsubscribeScene(prevProps.activeScene._id);
+            HubSendActions.subscribeScene(this.props.activeScene._id);
```

Comment:
I've moved how the Scene Player subscribes for Scene updates, tested and this reduces the number of emit changes being pushed out due to over subscription.

Reply:
This allows the player only subscribe to the active scene being played out, in particular this is for the use case of playing the scene while authoring it.